extensions/ql-vscode/src/remote-queries/variant-analysis-manager.ts

```diff
@@ -176,6 +177,20 @@ export class VariantAnalysisManager extends DisposableObject implements VariantA
     return this.queue.pending;
   }
 
+  public getVariantAnalysisStorageLocation(variantAnalysisId: number): string {
+    return this.variantAnalysisResultsManager.getStorageDirectory(variantAnalysisId);
+  }
+
+  /**
+   * Prepares a directory for storing results for a variant analysis.
+   * This directory contains a timestamp file, which will be
+   * used by the query history manager to determine when the directory
+   * should be deleted.
+   */
+  public async prepareStorageDirectory(variantAnalysisId: number): Promise<void> {
```

Comment:
Follows the example of the remote queries manager:
https://github.com/github/vscode-codeql/blob/424520613e4d20526255e71514debb57fe553ede/extensions/ql-vscode/src/remote-queries/remote-queries-manager.ts#L285-L287